service_mesh/service_mesh_install/installing-ossm.adoc

```diff
@@ -4,52 +4,38 @@ include::modules/ossm-document-attributes.adoc[]
 :context: installing-ossm
 toc::[]
 
-Installing the {ProductShortName} involves installing the Operator, and then creating and managing a custom resource definition file to deploy the control plane.
+Installing the {ProductShortName} involves installing the Elasticsearch, Jaeger, Kiali and {ProductShortName} Operators, creating and managing a custom resource definition file to deploy the control plane, and creating a `ServiceMeshMemberRoll` to specify the namespaces associated with the {ProductShortName}.
```

Comment:
Replace "...a custom resource definition file to deploy..." with "...a `ServiceMeshControlPlane` resource to deploy...`.

Also: add "resource" after `ServiceMeshMemberRoll`.

Reply:
Fixed.